packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -97,27 +94,27 @@ export async function makeWallet(
     } = offerRules;
 
     const instanceHandleP = E(registrar).get(instanceId);
-    const regAssay0P = E(registrar).get(assayId0);
-    const regAssay1P = E(registrar).get(assayId1);
-    const purseAssay0P = E(purse0).getAssay();
-    const purseAssay1P = E(purse1).getAssay();
-    const purseUnit0P = E(purseAssay0P).makeUnits(extent0 || 0);
-    const purseUnit1P = E(purseAssay0P).makeUnits(extent1 || 0);
+    const regIssuer0P = E(registrar).get(issuerId0);
+    const regIssuer1P = E(registrar).get(issuerId1);
+    const purseBrand0P = E(purse0).getBrand();
+    const purseBrand1P = E(purse1).getBrand();
+    const purseUnit0P = E(purseBrand0P).makeUnits(extent0 || 0);
```

Comment:
`makeUnits` doesn't exist anymore, so I think we want to either do `amountMath.make` or make it by hand.

Reply:
Units -> Amount